src/sentry/lang/javascript/processor.py

```diff
@@ -536,6 +536,9 @@ def add_raw_stacktraces(self, data, release):
 
             raw_frames = []
             for frame in exc['stacktrace']['frames']:
+                if 'data' not in frame:
+                    continue
+
                 frame = frame['data']['raw']
```

Comment:
Then the guard around L557 isn't needed, I guess? Since the key exists, the `pop()` should never raise a `KeyError`.